### PR TITLE
Fix 5301: Enchance doc for django celery backend.

### LINKED ISSUE #5301: docs: undocumented use of CELERY_CACHE_BACKEND in django-celery-results
# Description

When using `django-celery-results` with the `CELERY_RESULT_BACKEND = 'django-cache'` setting, it is possible to use the `CELERY_CACHE_BACKEND` to specify a Django cache other than `default`.

This is undocumented. Furthermore, the Configuration page of the user guide [describes](http://docs.celeryproject.org/en/v4.2.0/userguide/configuration.html#std:setting-cache_backend) this setting as "no longer used", which appears to be false.

# Suggestions
Describe the use of `CELERY_CACHE_BACKEND` by `django-celery-results`.

### PR DESCRIPTION
**Pull request by sp1rs**
Fixes #5301

### CODE REVIEW

File: docs/django/first-steps-with-django.rst [hunk 1]
```diff
@@ -206,6 +206,22 @@ To use this with your project you need to follow these steps:
...
+
+    .. code-block:: python
+
+        # celery setting.
+        CELERY_CACHE_BACKEND = 'default'
+
+        # django setting.
+        CACHES = {
+            'default': {
+                'BACKEND': 'api.lib.redis_cache_backend.RedisCache',
```
Comment (auvipy) [inline docs/django/first-steps-with-django.rst]: api??

Comment (sp1rs) [reply docs/django/first-steps-with-django.rst]: Just a random path.

Comment (auvipy) [reply docs/django/first-steps-with-django.rst]: /path/to/cachebackend/DBbackend

Comment (sp1rs) [reply docs/django/first-steps-with-django.rst]: Done.

Comment (auvipy) [reply docs/django/first-steps-with-django.rst]: CACHES = {
    'default': {
        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
        'LOCATION': 'my_cache_table',
    }
}

got from https://docs.djangoproject.com/en/2.2/topics/cache/

Comment (sp1rs) [reply docs/django/first-steps-with-django.rst]: Done.

**Pull request merged**